### PR TITLE
Harden MRTK inspectors / Move MRTK addition to scene to utility class

### PR DESCRIPTION
**Pull request by Railboy**
## Overview

This PR combines #4467 & #4435 and addresses comments from both.

### CODE REVIEW

File: Assets/MixedRealityToolkit/Inspectors/Utilities/MixedRealityInspectorUtility.cs [hunk 1]
```diff
@@ -47,13 +37,30 @@ public static bool CheckMixedRealityConfigured(bool renderEditorElements = true,
...
                 }
-
                 return false;
             }
 
             return true;
         }
 
+        /// <summary>
+        /// If MRTK is not initialized in scene, adds & initializes instance to current scene
```
Comment (wiwei) [inline Assets/MixedRealityToolkit/Inspectors/Utilities/MixedRealityInspectorUtility.cs]: >& [](start = 54, length = 1)

I think this character might throw off the docs build (I remember the docs build failing on this one:

[https://dev.azure.com/aipmr/MixedRealityToolkit-Unity-CI/_build/results?buildId=1633](https://dev.azure.com/aipmr/MixedRealityToolkit-Unity-CI/_build/results?buildId=1633))

I think that just making it "and" should be enough to get it good.

File: Assets/MixedRealityToolkit/Inspectors/Profiles/BaseMixedRealityToolkitConfigurationProfileInspector.cs [hunk 2]
```diff
@@ -130,73 +120,92 @@ protected bool DrawBacktrackProfileButton(string message, UnityEngine.Object act
...
             EditorGUILayout.BeginHorizontal();
                 EditorGUILayout.LabelField(new GUIContent(title, description), EditorStyles.boldLabel, GUILayout.ExpandWidth(true));
             EditorGUILayout.EndHorizontal();
 
             EditorGUILayout.LabelField(string.Empty, GUI.skin.horizontalSlider);
+        }
 
-            return true;
+        /// <summary>
+        /// If MRTK is in scene & input system is disabled, then show error message
```
Comment (wiwei) [inline Assets/MixedRealityToolkit/Inspectors/Profiles/BaseMixedRealityToolkitConfigurationProfileInspector.cs]:
```suggestion
        /// If MRTK is in scene and input system is disabled, then show error message
```

Comment (wiwei) [reply Assets/MixedRealityToolkit/Inspectors/Profiles/BaseMixedRealityToolkitConfigurationProfileInspector.cs]: I think there's one more docs issue

**Comment by keveleigh** [comment]
>This PR combines #4467 & #4435 and addresses comments from both.

@railboy should those PRs be closed now, then?

**Comment by Railboy** [comment]
@keveleigh Yup they can be closed.

**Pull request merged**